src/plugins/intel_cpu/src/memory_desc/cpu_memory_desc_utils.h

```diff
@@ -102,6 +102,8 @@ class MemoryDescUtils {
      * @return dims as string
      */
     static std::string dims2str(const VectorDims& dims);
+
+    static std::string computeWeightsStringHash(const MemoryCPtr memory, const std::shared_ptr<DnnlMemoryDesc> dstDesc);
```

Comment:
Please add brief description of the parameters and func behavior

Reply:
Done, thanks!